lib/liquid/standardfilters.rb

```diff
@@ -335,6 +336,11 @@ def floor(input)
       raise Liquid::FloatDomainError, e.message
     end
 
+    def hmac_sha256(input, secret)
+      digest = OpenSSL::Digest.new('sha256')
+      OpenSSL::HMAC.hexdigest(digest, secret, input)
```

Comment:
Does this handle the case where the input or secret is nil? Usually in Liquid we allow nil values in filters and return nil.